aws/dms/tasks.yml

```diff
@@ -47,7 +47,8 @@ cron:
 - dashboard.levels
 - dashboard.script_levels
 - dashboard.levels_script_levels
-- dashboard.courses
+- dashboard.unit_groups
+    rename: courses
```

Comment:
Syntax taken from here: https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Tasks.CustomizingTasks.TableMapping.html#CHAP_Tasks.CustomizingTasks.TableMapping.SelectionTransformation.Transformations

Reply:
I think the `rename` property here won't trigger creation of a DMS `rename` transformation rule until we update `[Cdo::DMS.tasks](https://github.com/code-dot-org/code-dot-org/blob/ea7bac4ecd268d886943a59dc6a3da0f90387377/lib/cdo/aws/dms.rb#L13)`.

Reply:
Hi @sureshc , does this seem like something you'll be able to get to this week? If not, I can take a crack at it.